src/test/java/seedu/address/logic/commands/GemCommandTest.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static seedu.address.logic.commands.GemCommand.MESSAGE_SUCCESS;
+import static seedu.address.logic.commands.RequiredCommandTest.MISSING_MODULE_1;
+import static seedu.address.logic.commands.RequiredCommandTest.SINGLE_MODULE_PATH;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.ReadOnlyGradPad;
+import seedu.address.model.module.Module;
+import seedu.address.storage.GemCommandStorage;
+import seedu.address.storage.JsonGradPadStorage;
+
+class GemCommandTest {
+    public static final Path COMPILED_PATH_1 = Paths.get("src/test/data/GemCommandTest/compiledsem1GEmodules.json");
+    public static final Path COMPILED_PATH_2 = Paths.get("src/test/data/GemCommandTest/compiledsem2GEmodules.json");
+    public static final String TEST_GEH_SEM1_PATH = "src/main/resources/data/GEM/GEHs1.json";
+    public static final String TEST_GEH_SEM2_PATH = "src/main/resources/data/GEM/GEHsem2.json";
```

Comment:
Nit: Maybe we can make these `Path` objects just like the two paths above it? It's a bit funny when the 4 variable names here say "path" but actually it's 2 paths and 2 strings HAHA

Reply:
ok HAHAHA standardised across